lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1724,11 +1731,16 @@ def _resource_sku_to_capability(  # noqa: C901
         azure_raw_capabilities: Dict[str, str] = {}
         if resource_sku.location_info:
             for location_info in resource_sku.location_info:
+                azure_raw_capabilities["availability_zones"] = location_info.zones
```

Comment:
The logic going to more complex. Can you use a simple json example to explain the structure in comment?

Reply:
Adding the example:
"locationInfo": [
    {
    "location": "southcentralus",
    "zoneDetails": [
        {
        "Name": [
            "3"
        ],
        "capabilities": [
            {
            "name": "UltraSSDAvailable",
            "value": "True"
            }
        ],
        "name": null
        }
    ],
    "zones": [
        "2",
        "3",
        "1"
    ]
    }
],